core/controllers/resources_test.py

```diff
@@ -243,8 +243,7 @@ class AssetDevHandlerAudioTest(test_utils.GenericTestBase):
     """Test the upload of audio files to GCS."""
 
     TEST_AUDIO_FILE_MP3 = 'cafe.mp3'
-    TEST_AUDIO_FILE_FLAC = 'cafe.flac'
-    TEST_AUDIO_FILE_FLAC_1 = 'galway.flac'
+    TEST_AUDIO_FILE_FLAC = 'galway.flac'
```

Comment:
If you're using a flac file can you use the existing cafe.flac here? I don't know where galway.flac is from and what license we can release it under.

Reply:
The exploration file system can only save files with size <= 1MB. That's why I chose galway.flac as its size is lesser than 1 MB.

I have got this license under the download page: 
Galway Kevin MacLeod (incompetech.com)
Licensed under Creative Commons: By Attribution 3.0 License
http://creativecommons.org/licenses/by/3.0/

Download link: https://incompetech.com/music/royalty-free/index.html?isrc=USUAN1700016

Reply:
But this PR doesn't seem to be doing the stuff that the license link you sent says it should do.

You might want to do that; alternatively, you could take the existing .flac file and truncate it so it's under 1 MB. That file is already under Apache 2, so I think that might be easier.

Reply:
Or can I use self.swap with MAX_FILE_SIZE_BYTES to make it less than 1 MB?

Reply:
That seems a bit weird. The file size is the file size, so I'm not sure what swapping it out means conceptually, and it seems easy enough to get around this the "normal" way. In general, prefer not using mocks where possible.

Reply:
Done